.github/workflows/main.yaml

```
@@ -1,20 +1,30 @@
 name: build
 
-on: [push, pull_request]
+on:
+  push:
+    paths:
+    - 'packages/**'
+    - '!packages/*/example/**'
```

```suggestion
    - 'performance_tests/**'
```

We don't need this anymore. Plus we forgot including this folder. :)

Protip: you can just click in the PR "accept suggestion" to speed up the process. ;)